modules/examples/src/main/java/UpdateDoc.java

```diff
@@ -53,31 +58,55 @@ public static void main(String[] args) {
 
             // Update the document in the database
             PostDocumentOptions updateDocumentOptions =
-                    new PostDocumentOptions.Builder()
-                            .db(exampleDbName)
-                            .document(document)
-                            .build();
+                new PostDocumentOptions.Builder()
+                    .db(exampleDbName)
+                    .document(document)
+                    .build();
 
+            // ================================================================
             // Note: for request byte stream use:
-            // PostDocumentOptions updateDocumentOptions =
-            //        new PostDocumentOptions.Builder()
-            //                .db(exampleDbName)
-            //                .contentType("application/json")
-            //                .body(documentAsByteStream)
-            //                .build();
+            /*
+            PostDocumentOptions updateDocumentOptions =
+                new PostDocumentOptions.Builder()
+                    .db(exampleDbName)
+                    .contentType("application/json")
+                    .body(documentAsByteStream)
+                    .build();
+            */
+            // ================================================================
 
             DocumentResult updateDocumentResponse = client
-                    .postDocument(updateDocumentOptions)
-                    .execute()
-                    .getResult();
+                .postDocument(updateDocumentOptions)
+                .execute()
+                .getResult();
+
+            // ====================================================================
+            // Note: updating the document can also be done with the "putDocument"
+            // method. docId and rev are required for an UPDATE operation:
```

Comment:
```suggestion
            // method. docId and rev are required for an UPDATE operation,
```

Reply:
Done in [2e6929d](https://github.com/IBM/cloudant-java-sdk/pull/247/commits/2e6929d1238d0c92191aa2103bc31965d045af49)